README.md

```diff
@@ -620,12 +620,12 @@ docker run \
 
 **Notes:**
 
-* The volume `/var/run/libvirt/libvirt-sock` is mandatory if you want to run Terrible locally (a local QEMU/KVM instance). In this way you directly interact with the QEMU/KVM api, provided by the system.
+* The volume `/var/run/libvirt/libvirt-sock` is mandatory if you want to run Terrible locally (a local QEMU/KVM instance). In this way you directly interact with the QEMU/KVM api, provided by the system (required).
 
-* The volume `./inventory-test.yml` is to include the inventory inside the container, to deploy the infrastructure.
+* The volume `./inventory-test.yml` is to include the inventory inside the container, to deploy the infrastructure (optional).
```

Comment:
this instead is required, without inventory how does this work?

Reply:
you can also write it inside the container :)

Reply:
Not really the type of use we're advertising here isn't it?